contrib/completions/zsh/oc

```diff
@@ -1,4 +1,4 @@
-#compdef kubectl
+#compdef oc
```

Comment:
This is generated file, so you can't switch this, you need to find a different place where that can be fixed.

Reply:
yeah just realized.

Reply:
is there any template from where this file is generated. I tried to look around but didn't find any.